channels/applicationinsights-channel-js/src/EnvelopeCreator.ts

```diff
@@ -22,367 +22,334 @@ function _setValueIf<T>(target:T, field:keyof T, value:any) {
     return setValue(target, field, value, isTruthy);
 }
 
-export abstract class EnvelopeCreator {
-    public static Version = "2.7.0-beta.1";
-
-    protected static extractPropsAndMeasurements(data: { [key: string]: any }, properties: { [key: string]: any }, measurements: { [key: string]: any }) {
-        if (!isNullOrUndefined(data)) {
-            objForEachKey(data, (key, value) => {
-                if (isNumber(value)) {
-                    measurements[key] = value;
-                } else if (isString(value)) {
-                    properties[key] = value;
-                } else if (hasJSON()) {
-                    properties[key] = getJSON().stringify(value);
-                }
-            });
-        }
+/*
```

Comment:
All minification improvements -- removing the internal class so the referenced names can be minified.